design/www/app/contacts/createprivatechatroom.factory.js

```diff
@@ -0,0 +1,11 @@
+(function() {
+   'use strict';
+   angular.module('starter')
+       .factory('createPrivateChatroomFactory', createPrivateChatroomFactory);
+
+   function createPrivateChatroomFactory($resource, Configurations) {
+       return $resource(Configurations.api_url+'/rooms/create_private_room?', {
+       	access_token:'@access_token'
```

Comment:
move access token to httpFactory.
after login all api should automatically have access token